npm/nameSpaceController_test.go

```diff
@@ -124,15 +125,24 @@ func updateNamespace(t *testing.T, f *nameSpaceFixture, oldNsObj, newNsObj *core
 	f.nsController.processNextWorkItem()
 }
 
-func deleteNamespace(t *testing.T, f *nameSpaceFixture, nsObj *corev1.Namespace) {
+func deleteNamespace(t *testing.T, f *nameSpaceFixture, nsObj *corev1.Namespace, isDeletedFinalStateUnknownObject IsDeletedFinalStateUnknownObject) {
```

Comment:
Potential function name ambiguity, can clarify between this helper deleteNamespace and f.nsController.deleteNamespace()

Reply:
Good point. I think last time Vamsi also mentioned this. We can discuss and refactor this point when we add more UTs with ipset and iptables.